collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
```

Comment:
If I understand correctly this should end with the suffix `_total`, as this is the count of devices available, correct? That suffix is Prometheus metric naming convention. https://prometheus.io/docs/practices/naming/